Content.Server/Xenoarchaeology/Equipment/Components/ArtifactAnalyzerComponent.cs

```diff
@@ -25,7 +25,7 @@ public sealed partial class ArtifactAnalyzerComponent : Component
     /// Each is 150 and added to this, so
     /// 550 / 700 / 850 / 1000
```

Comment:
update this too as its outdated

Reply:
this comment feels bogus overall like 400 wasnt in that before but sure

Reply:
400 + 150, 400 + 300, 400 + 350

why the comment says this idk